extensions/azurecore/src/azureResource/providers/database/databaseService.ts

```diff
@@ -19,18 +20,27 @@ export class AzureResourceDatabaseService implements IAzureResourceService<azure
 		const resourceClient = new ResourceGraphClient(credential, { baseUri: account.properties.providerSettings.settings.armResource.endpoint });
 
 		// Query servers and databases in parallel (start both promises before waiting on the 1st)
-		let serverQueryPromise = queryGraphResources<GraphData>(resourceClient, subscriptions, serversQuery);
+		let synapseQueryPromise = queryGraphResources<GraphData>(resourceClient, subscriptions, synapseWorkspacesQuery);
+		let serverQueryPromise = queryGraphResources<GraphData>(resourceClient, subscriptions, sqlServersQuery);
 		let dbQueryPromise = queryGraphResources<GraphData>(resourceClient, subscriptions, `where type == "${azureResource.AzureResourceType.sqlDatabase}"`);
-		let servers: DbServerGraphData[] = await serverQueryPromise as DbServerGraphData[];
+		let server1: (DbServerGraphData | DbSynapseGraphData)[] = await serverQueryPromise as DbServerGraphData[];
```

Comment:
1. This should just be `DbServerGraphData[]`. 

2. If you just do this above

`let serverQueryPromise = queryGraphResources<DbServerGraphData>(resourceClient, subscriptions, sqlServersQuery);` 

That should save you from having to cast it here. 
3. This should just be `servers` for the variable name

Reply:
I need to merge this with the synapseWorkspaces, together, so a separate variable is needed below.